docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -19,6 +19,19 @@ cluster to Teleport.
 
 (!docs/pages/kubernetes-access/helm/includes/teleport-cluster-prereqs.mdx!)
 
+## Choose a Kubernetes namespace and Helm release name
+
+<Admonition type="note">
```

Comment:
I think this is an interesting idea. When would users tend to change the namespace and release name from `teleport`?

Reply:
Generally it'd be the case if they're installing into a cluster which is being used for multiple Teleport installations, or if they have restrictions on their deployments imposed by cluster admins/security.